resources/migrations/000_migrations.yaml

```diff
@@ -14788,7 +14788,7 @@ databaseChangeLog:
       comment: Added 0.47.0 - Add join-alias to the report_card.visualization_settings.column_settings field refs
       changes:
         - customChange:
-            class: "metabase.db.custom_migrations.AddJoinAliasToVisualizationSettingsFieldRefs"
+            class: "metabase.db.custom_migrations.AddJoinAliasToColumnSettingsFieldRefs"
```

Comment:
I renamed this, changing `VisualizationSettings` to `ColumnSettings` for consistency and correctness. 

Consistency: there is another custom migration called `MigrateLegacyColumnSettingsFieldRefs` which updates the same keys

Correctness: `visualization_settings` can contain field refs elsewhere in the map, but this migration only touches those in `visualization_settings.column_settings`.